[Role][1625]simple_role_to_get_job_status (#2386)
ansible-collections/ibm_zos_core: roles/job_status/README.md

# ibm.ibm_zos_core.job_status

> This role gathers job status and if the jos is active about a giving a job_id.

Reviewer: Would suggest a different phrasing 

```suggestion
This role gathers job status information and checks if a job is active, given a specific job ID.
```
